src/pathToVictory/services/pathToVictory.service.ts

```diff
@@ -352,24 +332,51 @@ export class PathToVictoryService extends createPrismaBase(
         message: candidateSlackMessage + debugMessage,
         channel: SlackChannel.botPathToVictoryIssues,
       })
+      // Mark as Failed
+      await this.completePathToVictory(
+        campaign.slug,
+        pathToVictoryResponse,
+        false,
+        P2VStatus.failed,
+        {
+          officeName,
+          electionLevel,
+          electionState,
+          electionCounty,
+          electionMunicipality,
+          subAreaName,
+          subAreaValue,
+          positionId,
+        },
+      )
+      await this.crmService.handleUpdateCampaign(
+        campaign,
+        'path_to_victory_status',
+        P2VStatus.failed,
+      )
+      return false
     }
-
-    await this.crmService.handleUpdateCampaign(
-      campaign,
-      'path_to_victory_status',
-      'Waiting',
-    )
-    return false
   }
 
   async completePathToVictory(
     slug: string,
     pathToVictoryResponse: {
-      counts: VoterCounts
+      counts: P2VCounts
       electionType: string
       electionLocation: string
     },
     sendEmail = true,
+    p2vStatusOverride?: P2VStatus,
+    officeContext?: {
+      officeName: string
+      electionLevel: string
+      electionState: string
+      electionCounty: string
+      electionMunicipality: string
+      subAreaName?: string
+      subAreaValue?: string
+      positionId?: string
+    },
```

Comment:
Please don't just keep creating this object over and over again.. define it once and just pass it into the method calls..